### ISSUE TITLE
Returned data is unhelpful

### ISSUE THREAD

**Issue by ArtskydJ** [TehShrike/noddity-renderer#6]
Right now, the rewrite branch of this module returns something like this
```html
blah blah words
<span class='noddity-template' data-noddity-post-file-name='do.md' data-noddity-template-arguments='{"1":"parse","2":"me"}' data-noddity-partial-name='9fee4400-62c0-11e5-a535-3ce89d6e46f5'>{{>9fee4400-62c0-11e5-a535-3ce89d6e46f5}}</span>
blah blah more words
<span class='noddity-template' data-noddity-post-file-name='do.md' data-noddity-template-arguments='{"1":"parse","2":"me"}' data-noddity-partial-name='9fee4400-62c0-11e5-a535-3ce89d6e46f5'>{{>9fee4400-62c0-11e5-a535-3ce89d6e46f5}}</span>
hooray words
```
To use this, it needs to be parsed again. :confused: 

Why not return something like this:
```js
[
  'blah blah words',
  {
    filename: 'do.md',
    arguments: {1: 'parse', 2: 'me' }
  },
  'blah blah more words',
  {
    filename: 'do.md',
    arguments: {1: 'parse', 2: 'me' }
  },
  'hooray words'
]
```
or this
```js
{
  words: [
    'blah blah words',
    'blah blah more words',
    'hooray words'
  ],
  templates: [
    {
      filename: 'do.md',
      arguments: {1: 'parse', 2: 'me' }
    },
    {
      filename: 'do.md',
      arguments: {1: 'parse', 2: 'me' }
    }
  ]
}
```

**Comment by ArtskydJ** [comment]
Probably this:
```js
[
  {
    type: 'string',
    value: 'markdown goes here'
  },
  {
    type: 'template',
    filename: 'whatever.md',
    arguments: { 1: 'parse', 2: 'me' }
  },
  {
    type: 'string',
    value: 'markdown goes here'
  }
]
```

**Comment by TehShrike** [comment]
Yeah, that seems reasonable to me.

**Comment by TehShrike** [comment]
Fixed by #5
